--- closure/bin/build/closurebuilder.py.orig	2015-11-12 11:25:11.000000000 +0100
+++ closure/bin/build/closurebuilder.py	2015-11-12 11:25:21.000000000 +0100
@@ -195,7 +195,7 @@
 
 def main():
   logging.basicConfig(format=(sys.argv[0] + ': %(message)s'),
-                      level=logging.INFO)
+                      level=logging.WARNING)
   options, args = _GetOptionsParser().parse_args()
 
   # Make our output pipe.
