diff -ruN goog.clean/array/array.js goog/array/array.js
--- goog.clean/array/array.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/array/array.js	2015-02-24 10:43:37.000000000 +0100
@@ -15,6 +15,7 @@
 /**
  * @fileoverview Utilities for manipulating arrays.
  *
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 
diff -ruN goog.clean/asserts/asserts.js goog/asserts/asserts.js
--- goog.clean/asserts/asserts.js	2015-11-12 11:09:36.000000000 +0100
+++ goog/asserts/asserts.js	2015-11-12 11:11:27.000000000 +0100
@@ -16,6 +16,8 @@
  * @fileoverview Utilities to check the preconditions, postconditions and
  * invariants runtime.
  *
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
+ *
  * Methods in this package should be given special treatment by the compiler
  * for type-inference. For example, <code>goog.asserts.assert(foo)</code>
  * will restrict <code>foo</code> to a truthy value.
diff -ruN goog.clean/base.js goog/base.js
--- goog.clean/base.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/base.js	2015-02-24 10:43:37.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Bootstrap for the Google JS Library (Closure).
+ * @suppress {reportUnknownTypes,es5Strict} DJM/SpinPunch
  *
  * In uncompiled mode base.js will write out Closure's deps file, unless the
  * global <code>CLOSURE_NO_DEPS</code> is set to true.  This allows projects to
diff -ruN goog.clean/crypt/base64.js goog/crypt/base64.js
--- goog.clean/crypt/base64.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/crypt/base64.js	2015-02-25 09:12:16.000000000 +0100
@@ -16,6 +16,7 @@
  * @fileoverview Base64 en/decoding. Not much to say here except that we
  * work with decoded values in arrays of bytes. By "byte" I mean a number
  * in [0, 255].
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * @author doughtie@google.com (Gavin Doughtie)
  * @author fschneider@google.com (Fritz Schneider)
diff -ruN goog.clean/crypt/crypt.js goog/crypt/crypt.js
--- goog.clean/crypt/crypt.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/crypt/crypt.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Namespace with crypto related helper functions.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 goog.provide('goog.crypt');
diff -ruN goog.clean/debug/debug.js goog/debug/debug.js
--- goog.clean/debug/debug.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/debug/debug.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Logging and debugging utilities.
+ * @suppress {reportUnknownTypes,es5Strict} DJM/SpinPunch
  *
  * @see ../demos/debug.html
  */
diff -ruN goog.clean/debug/logbuffer.js goog/debug/logbuffer.js
--- goog.clean/debug/logbuffer.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/debug/logbuffer.js	2015-02-25 09:12:16.000000000 +0100
@@ -20,6 +20,7 @@
  * log records and expect that they are not overwriten at a later point.
  *
  * @author agrieve@google.com (Andrew Grieve)
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 goog.provide('goog.debug.LogBuffer');
diff -ruN goog.clean/debug/logger.js goog/debug/logger.js
--- goog.clean/debug/logger.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/debug/logger.js	2015-02-25 09:12:16.000000000 +0100
@@ -16,6 +16,7 @@
  * @fileoverview Definition of the Logger class. Please minimize dependencies
  * this file has on other closure classes as any dependency it takes won't be
  * able to use the logging infrastructure.
+ * @suppress {reportUnknownTypes,es5Strict} DJM/SpinPunch
  *
  * @see ../demos/debug.html
  */
diff -ruN goog.clean/disposable/disposable.js goog/disposable/disposable.js
--- goog.clean/disposable/disposable.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/disposable/disposable.js	2015-02-25 09:12:16.000000000 +0100
@@ -16,6 +16,7 @@
  * @fileoverview Implements the disposable interface. The dispose method is used
  * to clean up references and resources.
  * @author arv@google.com (Erik Arvidsson)
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 
diff -ruN goog.clean/events/browserevent.js goog/events/browserevent.js
--- goog.clean/events/browserevent.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/events/browserevent.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview A patched, standardized event object for browser events.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * <pre>
  * The patched event object contains the following members:
diff -ruN goog.clean/events/browserfeature.js goog/events/browserfeature.js
--- goog.clean/events/browserfeature.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/events/browserfeature.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Browser capability checks for the events package.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  */
 
diff -ruN goog.clean/events/events.js goog/events/events.js
--- goog.clean/events/events.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/events/events.js	2015-02-25 09:12:16.000000000 +0100
@@ -17,6 +17,7 @@
  * targets and custom JavaScript event targets
  * ({@code goog.events.Listenable}). This provides an abstraction
  * over browsers' event systems.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * It also provides a simulation of W3C event model's capture phase in
  * Internet Explorer (IE 8 and below). Caveat: the simulation does not
diff -ruN goog.clean/events/eventtarget.js goog/events/eventtarget.js
--- goog.clean/events/eventtarget.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/events/eventtarget.js	2015-02-25 09:12:16.000000000 +0100
@@ -16,6 +16,7 @@
  * @fileoverview A disposable implementation of a custom
  * listenable/event target. See also: documentation for
  * {@code goog.events.Listenable}.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * @author arv@google.com (Erik Arvidsson) [Original implementation]
  * @author pupius@google.com (Daniel Pupius) [Port to use goog.events]
diff -ruN goog.clean/events/listenable.js goog/events/listenable.js
--- goog.clean/events/listenable.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/events/listenable.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview An interface for a listenable JavaScript object.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 goog.provide('goog.events.Listenable');
diff -ruN goog.clean/events/listenermap.js goog/events/listenermap.js
--- goog.clean/events/listenermap.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/events/listenermap.js	2015-02-25 09:12:16.000000000 +0100
@@ -16,6 +16,7 @@
  * @fileoverview A map of listeners that provides utility functions to
  * deal with listeners on an event target. Used by
  * {@code goog.events.EventTarget}.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * WARNING: Do not use this class from outside goog.events package.
  *
diff -ruN goog.clean/iter/iter.js goog/iter/iter.js
--- goog.clean/iter/iter.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/iter/iter.js	2015-02-25 09:12:16.000000000 +0100
@@ -15,6 +15,7 @@
 /**
  * @fileoverview Python style iteration utilities.
  * @author arv@google.com (Erik Arvidsson)
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 
diff -ruN goog.clean/json/json.js goog/json/json.js
--- goog.clean/json/json.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/json/json.js	2015-02-25 09:12:16.000000000 +0100
@@ -15,6 +15,7 @@
 /**
  * @fileoverview JSON utility functions.
  * @author arv@google.com (Erik Arvidsson)
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 
diff -ruN goog.clean/net/xhrio.js goog/net/xhrio.js
--- goog.clean/net/xhrio.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/net/xhrio.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Wrapper class for handling XmlHttpRequests.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * One off requests can be sent through goog.net.XhrIo.send() or an
  * instance can be created to send multiple requests.  Each request uses its
@@ -478,7 +479,8 @@
   try {
     goog.log.fine(this.logger_, this.formatMsg_('Opening Xhr'));
     this.inOpen_ = true;
-    this.xhr_.open(method, url, true);  // Always async!
+    var url2 = /** @type {string} */ (url); // DJM/SpinPunch
+    this.xhr_.open(method, url2, true);  // Always async!
     this.inOpen_ = false;
   } catch (err) {
     goog.log.fine(this.logger_,
diff -ruN goog.clean/net/xmlhttp.js goog/net/xmlhttp.js
--- goog.clean/net/xmlhttp.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/net/xmlhttp.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Low level handling of XMLHttpRequest.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  * @author arv@google.com (Erik Arvidsson)
  * @author dbk@google.com (David Barrett-Kahn)
  */
diff -ruN goog.clean/object/object.js goog/object/object.js
--- goog.clean/object/object.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/object/object.js	2015-02-24 10:43:37.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Utilities for manipulating objects/maps/hashes.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 goog.provide('goog.object');
diff -ruN goog.clean/reflect/reflect.js goog/reflect/reflect.js
--- goog.clean/reflect/reflect.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/reflect/reflect.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Useful compiler idioms.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  */
 
diff -ruN goog.clean/string/string.js goog/string/string.js
--- goog.clean/string/string.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/string/string.js	2015-02-24 10:43:37.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Utilities for string manipulation.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  */
 
 
diff -ruN goog.clean/structs/map.js goog/structs/map.js
--- goog.clean/structs/map.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/structs/map.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Datastructure: Hash Map.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * @author arv@google.com (Erik Arvidsson)
  * @author jonp@google.com (Jon Perlow) Optimized for IE6
diff -ruN goog.clean/structs/set.js goog/structs/set.js
--- goog.clean/structs/set.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/structs/set.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Datastructure: Set.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * @author arv@google.com (Erik Arvidsson)
  * @author pallosp@google.com (Peter Pallos)
diff -ruN goog.clean/structs/structs.js goog/structs/structs.js
--- goog.clean/structs/structs.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/structs/structs.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Generics method for collection-like classes and objects.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * @author arv@google.com (Erik Arvidsson)
  *
diff -ruN goog.clean/timer/timer.js goog/timer/timer.js
--- goog.clean/timer/timer.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/timer/timer.js	2015-02-25 09:12:16.000000000 +0100
@@ -15,6 +15,7 @@
 /**
  * @fileoverview A timer class to which other classes and objects can
  * listen on.  This is only an abstraction above setInterval.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * @see ../demos/timers.html
  */
diff -ruN goog.clean/uri/utils.js goog/uri/utils.js
--- goog.clean/uri/utils.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/uri/utils.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Simple utilities for dealing with URI strings.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  *
  * This is intended to be a lightweight alternative to constructing goog.Uri
  * objects.  Whereas goog.Uri adds several kilobytes to the binary regardless
diff -ruN goog.clean/useragent/useragent.js goog/useragent/useragent.js
--- goog.clean/useragent/useragent.js	2015-11-12 11:10:08.000000000 +0100
+++ goog/useragent/useragent.js	2015-02-25 09:12:16.000000000 +0100
@@ -14,6 +14,7 @@
 
 /**
  * @fileoverview Rendering engine detection.
+ * @suppress {reportUnknownTypes} DJM/SpinPunch
  * @see <a href="http://www.useragentstring.com/">User agent strings</a>
  * For information on the browser brand (such as Safari versus Chrome), see
  * goog.userAgent.product.
